src/main/java/ome/logic/ConfigImpl.java

```diff
@@ -247,7 +247,7 @@ public String getConfigValue(String key) {
     @RolesAllowed("system")
     public Map<String, String> getConfigDefaults() {
         File etc = new File("etc");
-        File omero = new File(etc, "omero.properties");
```

Comment:
I don't think this one will work. Here we will still need the aggregator code.

Reply:
I have few more commits to push